.sonarqube/sonar-scanner.properties

```diff
@@ -153,8 +152,8 @@ carma_wm.sonar.sources                  = src
 carma_wm.sonar.exclusions  =test/**
 carma_wm_ctrl.sonar.sources             = src
 carma_wm_ctrl.sonar.exclusions  =test/**
-mpc_follower_wrapper.sonar.sources      = src
-mpc_follower_wrapper.sonar.exclusions  =test/**
+# mpc_follower_wrapper.sonar.sources      = src
```

Comment:
Instead of commenting these out, I think we can remove these lines completely for `mpc_follower_wrapper`

Reply:
Removed unwanted sonar modules